pkg/controller/daemon/daemon_controller_test.go

```diff
@@ -2053,6 +2103,7 @@ func TestUpdateNode(t *testing.T) {
 		ds                 *apps.DaemonSet
 		expectedEventsFunc func(strategyType apps.DaemonSetUpdateStrategyType) int
```

Comment:
This is such an odd pattern. Clearly update strategy and `expectedEvents int` should be fields of the test struct. Out of scope for this PR, but I finding these tests difficult to read because of patterns like this.

Reply:
Sure, I can create a separate issue for this and it could be included in table driven tests issue that you created.

Reply:
I guess DS controller is technically owned by sig-scheduling... :smile:

Reply:
I think by sig-apps but we are sig-scheduling is partly responsible as well.